### DISCUSSION TITLE
Follow without account

### DISCUSSION THREAD

**Discussion by yamarten** [bluesky-social/atproto#821] [How to do it?] [answered]
I want to subscribe to the posts without creating an account and relying on PDS. For example, it seems this can be realized by calling `getRepo` of the owner's PDS directly from the client.

Is such usage allowed in ATP? If not, what are the factors that prohibit it?

**Comment by dholms** [comment, marked as answer]
The API Is open so this feature is allowed, however, it is not supported in our application.

We'll be creating a public webapp soon that does not require an account (similar to other logged-out social media). From there you will be able to browse an author's feed. Though a "follow" is a relationship between two accounts, therefore requiring you to have an account.

In the meantime, I'd point you to `com.atproto.repo.listRecords` instead of `getRepo`. It should be lighterwight, easier to parse & you can grab only the records you're interested in (such as posts).

**Reply by yamarten** [reply]
It's great to be able to subscribe by a client like a feed reader, even if it's not supported.

As you said, `listRecords` is a more practical and worth trying. thank you!
